src/connect/components/drive-settings-modal/drive-settings-modal.stories.tsx

```diff
@@ -50,12 +49,12 @@ export const Default: Story = {
                     {...args}
                     modalProps={{
                         ...args.modalProps,
-                        onClose: () => {
+                        onOpenChange: open => {
```

Comment:
The main difference between this dialog library and the existing one is that it uses `onOpenChange` instead of `onClose`. I actually prefer this because you can just pass your `setOpen` function as-is and it will keep the state in sync.